screens/Settings/SettingScreenController.ts

```diff
@@ -138,14 +138,18 @@ export function useSettingsScreen(props: RootRouteProps) {
         )
       ),
 
+    RECEIVE_CARD: () => {
+      props.navigation.navigate('Request');
+      setIsVisible(false);
+    },
+
     TOGGLE_BIOMETRIC: (enable: boolean) =>
       settingsService.send(SettingsEvents.TOGGLE_BIOMETRIC_UNLOCK(enable)),
 
     LOGOUT: () => {
       setIsVisible(false);
       const navigate = () => {
         authService.send(AuthEvents.LOGOUT());
-        props.navigation.navigate('Welcome');
```

Comment:
@tilak-puli Is this line required? Did this prevent the iOS app from crashing?

I had wrongly suggested to remove this line in the past PR.

Reply:
Okay, my bad. I had suggested to remove the line which was navigating to the request screen

Reply:
This was done by @srikanth716. I confirmed with him before removing the line. That would never affect the IOS app.

Reply:
Yes, @srikanth716 is right here. I tested this on the iOS physical phone. It worked well as authService sent the `LOGOUT` event which brought the app.ts state machine to `unauthorized` state which made sure that AppLayout.tsx showed the correct screen as per it's logic. 👍🏼

Reply:
Yeah, Srikanth is right here. Just got clarification from him.